sklearn/ensemble/tests/test_stacking.py

```diff
@@ -52,6 +55,9 @@
 X_diabetes, y_diabetes = diabetes.data, diabetes.target
 iris = load_iris()
 X_iris, y_iris = iris.data, iris.target
+X_multilabel_r, y_multilabel_r = make_multilabel_classification(
+    n_classes=3, random_state=42
+)
```

Comment:
I think that it would also be appropriate to test the binary classification case.